.bazelci/presubmit.yml

```diff
@@ -37,6 +37,17 @@ tasks:
       - "@go_sdk//..."
     test_targets:
       - "//..."
+  ubuntu2004_examples:
+    name: Examples test Ubuntu
+    platform: ubuntu2004
+    working_directory: examples/go-code-tutorial
+    shell_commands:
+      - "sudo apt-get update"
+      - "sudo apt-get install wamerican"
```

Comment:
this seems really surprising, can you add a comment here about why we have to install custom packages on the machine outside of Bazel? it makes it harder for users to reproduce the same result locally, doesn't it?

Reply:
You only have to install it if you are on LInux.  I am happy to take this as a bug and rewrite this.  We would need to change the source code and also the tutorial.

Reply:
I could generate a Fibonacci sequence or something.

Reply:
@alexeagle @fmeum any ideas on what package we should pull in? Also, should we fork the package, so that it does not disappear?

Reply:
I removed this and I am going to use klog